css/main.css

```diff
@@ -1,3 +1,58 @@
 /* Add your custom CSS here */
+*{
+    margin: 0px;
+}
+body {
+    font-family: "https://fonts.google.com/specimen/Quicksand";
+    color: rgb(0,0,0); 
+}
+
+.img-prof {
+    border-radius: 40%;
+}
+a {
+    padding:5px;
+    ;
+}
+
+.flex-nav{
+    display: flex;
+    justify-content: space-between;
+    padding: 5vh;
+}
+h1 h2 h3 h4 h5 h6 {
+    font-family: "https://fonts.google.com/specimen/Poppins";
+}
+.background-beige{
+    background-color: #E6DACD;
+    margin: 0px;
```

Comment:
Problem here: you are using this class on the same level as Bootstrap classes `col-md-5 col-12`. 
Whenever you apply your own custom class to an element that also has Boostrap grid classes, you **must not** include any left or right margin, padding or width, as it will interfer with how Bootstrap works. 
You can however keep using this class for the `background-color` and `height`, just remove the margin rule.